code/modules/clothing/suits/bio.dm

```diff
@@ -9,6 +9,7 @@
 	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEHAIR|HIDEFACIALHAIR|HIDEFACE
 	resistance_flags = ACID_PROOF
 	flags_cover = HEADCOVERSEYES | HEADCOVERSMOUTH
+	snug_fit = TRUE

```

Comment:
I don't think that's a good idea as well, since you can use these to protect yourself from things like xenos